doc/source/getting_started/install.rst

```diff
@@ -364,6 +387,8 @@ pymysql                   1.0.2              MySQL engine for sqlalchemy
 Other data sources
 ^^^^^^^^^^^^^^^^^^
 
+Can be managed with the ``hdf5, parquet, feather, spss, excel`` extras, depending on required compatibility.
```

Comment:
Would be good to add another column to specify which dependency is captured with which label.